app/views/layouts/_adv_search_footer.html.haml

```diff
@@ -26,7 +26,7 @@
                    :title   => t,
                    :onclick => "miqAjaxButton('#{url_for_only_path(:action => 'adv_search_button', :button => "apply")}');")
     - if @edit[@expkey][:selected] && @edit[@expkey][:selected][:typ] != "default" && @edit[@expkey][:selected][:id] != 0
-      - if admin_user? || @edit[@expkey][:selected][:typ] == "user"
+      - if report_admin_user? || @edit[@expkey][:selected][:typ] == "user"
```

Comment:
@kbrock same comment as above here, this shouldn't limited to a report_admin_user, this checkbox should have been available to super admin or an admin user

Reply:
@kbrock what is a method to check if a user is admin user ?

Reply:
(we spoke offline but answering for others)

We want to focus on features not roles.
features are mapped to roles out of the box, so it will look like we map functionality to roles, but mapping to features buys our customers a lot of flexibility.